DataBiosphere/leonardo
# [IA-4283] Use latest terra-azure-relay-listeners image

…3311)


__Jira ticket__: https://broadworkbench.atlassian.net/browse/IA-4283


Relies on tarl PR: https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/35

## Summary of changes



- Use latest terra-azure-relay-listeners image hash; this change fixes calls to updateDateAccessed.

## Testing these changes

### What to test

- Create a new Azure VM with autopause threshold 10 minutes
- [optional] Follow the steps outlined in https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/2726592515/Azure+VM+-+SSH to ssh into the VM to view logs (with `docker logs listener -f`)
- Open JupyterLab on your new VM and do some trivial coding.
- Wait 6 minutes
- Interact with your VM a bit.
- Wait 6 minutes
- Verify your VM has not stopped.
- Interact a bit with your VM again.
- Wait 10 minutes
- Verify your VM has stopped.

### Who tested and where

  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [x] I validated this change on my BEE
